src/gen-ragel-artifacts.py

```diff
@@ -4,22 +4,29 @@
 
 import os, os.path, sys, subprocess, shutil
 
+uninstalled_error = """
+'ragel' is missing on your system. In order to develop HarfBuzz itself,
+specifically, by editing the *.rl source files, you have to install ragel
+in order to transpile them to C++ headers.
+""".strip()
+
 ragel = sys.argv[1]
-if not ragel:
-	sys.exit ('You have to install ragel if you are going to develop HarfBuzz itself')
 
-if len (sys.argv) < 4:
-	sys.exit (__doc__)
+if len(sys.argv) != 5:
+	sys.exit(__doc__)
```

Comment:
Please keep the existing code style, we are using it consistently in this project.